flask_s3.py

```diff
@@ -39,6 +41,48 @@
 
 __version__ = (0, 2, 10)
 
+FLASKS3_PREFIX_DEFAULT = ''
+FLASKS3_VERSION_FUNCTION_DEFAULT = None
```

Comment:
We don't need these constants here, you can just use`app.config.get('FLASKS3_VERSION_FUNCTION', None)`